api/v1alpha1/openapi.yaml

```diff
@@ -5455,29 +5820,22 @@ components:
     AuthConfig:
       type: object
       properties:
-        authType:
+        apiVersion:
           type: string
-          description: Auth type.
-        authURL:
+          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources.'
+        providers:
+          type: array
+          description: List of all available authentication providers.
+          items:
+            $ref: '#/components/schemas/AuthProvider'
+        defaultProvider:
           type: string
-          description: Auth URL.
-        authOrganizationsConfig:
-          $ref: '#/components/schemas/AuthOrganizationsConfig'
-      required:
-        - authType
-        - authURL
-        - authOrganizationsConfig
-      description: Auth config.
-    AuthOrganizationsConfig:
-      type: object
-      description: Auth related organizations configuration.
-      properties:
-        enabled:
+          description: Name of the default authentication provider.
+        organizationsEnabled:
```

Comment:
Remove?

Reply:
lets do this in the removal of all org enable auth enable stuff